apps/wallet/src/shared/utils/index.ts

```diff
@@ -15,9 +16,22 @@ export function openInNewTab() {
 
 export function usePageView() {
     const location = useLocation();
+    const { apiEnv, customRPC } = useAppSelector((state) => state.app);
+    // Use customRPC url if apiEnv is customRPC
+    const activeNetwork =
+        customRPC && apiEnv === 'customRPC' ? customRPC : apiEnv;
+
     useEffect(() => {
         trackPageview({
             url: location.pathname,
         });
-    }, [location]);
+
+        // Send a network event to Plausible with the page and url params
+        trackEvent('PageByNetwork', {
+            props: {
+                name: activeNetwork,
```

Comment:
Should this also be `toUpperCase()` like in explorer?

Reply:
done